hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/config/CommonConsumerProperties.java

```diff
@@ -0,0 +1,155 @@
+package pl.allegro.tech.hermes.consumers.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import pl.allegro.tech.hermes.common.util.InetAddressInstanceIdResolver;
+import pl.allegro.tech.hermes.consumers.CommonConsumerParameters;
+
+@ConfigurationProperties(prefix = "consumer")
+public class CommonConsumerProperties {
+
+    private int threadPoolSize = 500;
+
+    private int inflightSize = 100;
+
+    private boolean filteringRateLimiterEnabled = true;
+
+    private int healthCheckPort = 8000;
+
+    private boolean filteringEnabled = true;
+
+    private long subscriptionIdsCacheRemovedExpireAfterAccessSeconds = 60L;
```

Comment:
What do you think about wrapping `supervisor`-related parameters into a dedicated class?